lms/djangoapps/discussion/rest_api/serializers.py

```diff
@@ -584,6 +586,10 @@ def get_abuse_flagged_any_user(self, obj):
         if _validate_privileged_access(self.context):
             return len(obj.get("abuse_flaggers", [])) > 0
 
+    def get_profile_image(self, obj):
+        request = self.context["request"]
+        return AccountLegacyProfileSerializer.get_profile_image(request.user.profile, request.user, request)
```

Comment:
I'm concerned about the coupling of serializers in the user_api djangoapp with apis in the discussions app.  I think it's pretty reasonable that we want to be able to send back profile image data from multiple places so I suggest that we extract the `get_profile_images` function to the [api.py](https://github.com/openedx/edx-platform/blob/master/openedx/core/djangoapps/user_api/accounts/api.py) file in the user_api django app so it's clear that others are expected to use this and that it forms an interface that should not break.

See https://github.com/openedx/edx-platform/blob/master/docs/decisions/0002-inter-app-apis.rst for the reasoning behind this suggestion.

Reply:
Done. I moved  `get_profile_images` method to api. Now we don't use method from AccountLegacyProfileSerializer in CommentSerializer.